fastai/widgets/class_confusion.py

```diff
@@ -0,0 +1,226 @@
+import math
+import pandas as pd
+import matplotlib.pyplot as plt
+
+from itertools import permutations, combinations
+from fastai.train import ClassificationInterpretation
+import ipywidgets as widgets
+
+class ClassLosses():
+    
+    """Plot the most confused datapoints and statistics for your misses. 
```

Comment:
Just give a one-line docstring here, then put the rest in the documentation of that widget (notebook to come after). Also, try to give a one-line docstring for each of the methods of this class (or make them private if the user isn't supposed to use them).